src/NServiceBus.Testing.Fakes/TestablePipelineContext.cs

```diff
@@ -42,8 +47,15 @@ public TestablePipelineContext(IMessageCreator messageCreator = null)
         /// <param name="options">The options for the send.</param>
         public virtual Task Send(object message, SendOptions options)
         {
-            sentMessages.Enqueue(new SentMessage<object>(message, options));
+            var headers = options.GetHeaders();
+
+            if (headers.ContainsKey(Headers.IsSagaTimeoutMessage) && 
+                headers[Headers.IsSagaTimeoutMessage] == bool.TrueString)
+            {
+                timeoutMessages.Enqueue(GetTimeoutMessage(message, options));
```

Comment:
was on the fence if it makes sense to add timeout messages into both collections. What's your thoughts? The problem is timeouts are also just normal sends. Having it in both gives the benefit that it remains compatible if you don't care about the actual timeout value. That's why I went with this approach.